gslides_api/agnostic/presentation.py

```
@@ -107,25 +133,26 @@ def from_markdown(
+                try:
+                    element = cls._create_element(
+                        name=element_name,
+                        content=content,  # Can be empty string, will become None
+                        content_type=content_type,
+                    )
+                    elements.append(element)
+                except ValueError as e:
+                    if on_invalid_element == "raise":
+                        raise ValueError(
+                            f"Invalid content for {content_type.value} element '{element_name}': {e}"
+                        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)